### PR TITLE
clean up gauth WIP

### PR DESCRIPTION
**Pull request by tduehr**

### CODE REVIEW

File: core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java [hunk 1]
```diff
@@ -45,9 +45,10 @@
         val appNames = new ArrayList<String>();
         val appName = casConfigurationPropertiesEnvironmentManager.getApplicationName();
         appNames.add("application");
-        appNames.add(appName.toLowerCase());
-        if (!appName.equals(appName.toLowerCase())) {
-            appNames.add(casConfigurationPropertiesEnvironmentManager.getApplicationName());
+        val appNameLower = appName.toLowerCase();
```
Comment (mmoayyed) [inline core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java]: This is fairly strange. Why aren't we using a Set?

Comment (mmoayyed) [reply core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java]: Actually, I take that back, but this block is fairly confusing!

Comment (tduehr) [reply core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java]: I blame spotbugs. Came up as a redundant method call. I don’t know why it’s getting flagged in this PR and not in master.

Comment (tduehr) [reply core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java]: I'll clean it up entirely... There's no reason to be doing it this way really...

Comment (tduehr) [reply core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java]: I take that back... no better way to do it.

**Pull request merged**